index.html

```diff
@@ -199,5 +199,77 @@ <h3 class="skill-title">Languages</h3>
         </ul>
       </div>
     </section>
+    <section id="contact-form">
+      <h3 id="contact-form-caption" class="heading text-secondary">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h3>
+      <form
+        action="https://formspree.io/f/mrgvbkea"
+        method="post"
+        id="contact-form"
+      >
+        <input
+          type="text"
+          max="30"
+          required
+          class="contact-input"
+          name="full-name"
+          placeholder="Full Name"
+        />
```

Comment:
- [ ] You did an excellent job working on this milestone! 💯 However, you misspelled the `max` property here. In order to make it work, it should be `maxlength` just as you wrote in the `<textarea>` element. Please fix this! 🤓 🔧